### ISSUE TITLE
Unable to Detect Open MATLAB Session (MacOS+R2018b)

### ISSUE THREAD

**Issue by pacarvalho** [OpenNFT/OpenNFT#37]
In MacOS 10.14.5 with MATLAB 64bit R2018b (9.5.0.1033004) the issue below occurs:

![OpenNFT_Issue_MATLAB_Closed](https://user-images.githubusercontent.com/860599/58950364-e8a8e780-878e-11e9-9e12-0f17e20faa80.gif)

I installed OpenNFT by running the commands in the README:

```bash
$ conda create -n OpenNFT_venv python=3.5 pip setuptools
$ conda activate OpenNFT_venv
(OpenNFT_venv)$ python -m pip install -U pip setuptools
(OpenNFT_venv)$ pip install git+https://github.com/OpenNFT/OpenNFT.git -v
```

I installed Matlab Python Engine running the commands below in bash:

```bash
cd "matlabroot/extern/engines/python"
python setup.py install
```

It does not appear OpenNFT can connect to the MATLAB session. Any ideas of what would cause this behavior or of how to solve it?

**Comment by ykoush** [comment]
Unfortunately on MacOS there is still unresolved issue for multiple matlab sessions. Please check the manual for this, there are half-automatic and automatic solutions (https://github.com/OpenNFT/OpenNFT/blob/master/macos_startmatlab.sh).

**Comment by pacarvalho** [comment]
@ykoush thank you for the quick reply. 

For some reason the issue appears to be with the `-nodesktop` flag for MATLAB2018b on MacOS. If I run your unmodified script (just updating it to reflect my version) it opens up the processes but the ones without a desktop stop themselves a few seconds later. If I update the script to run all instances with `-desktop` the software works. 

```bash
#!/bin/sh
#
# This bash script starts multiple Matlab on macos
# IMPORTANT: Specify the absolute path to appropriate Matlab with installed Matlab engine
# __________________________________________________________________________
# Copyright (C) 2016-2019 OpenNFT.org
#
/Applications/MATLAB_R2018b.app/bin/matlab -desktop -r "matlab.engine.shareEngine('MATLAB_NFB_MAIN_00001')" > /dev/null 2>&1 &
/Applications/MATLAB_R2018b.app/bin/matlab -desktop -r "matlab.engine.shareEngine('MATLAB_NFB_PTB_00001')" > /dev/null 2>&1 &
/Applications/MATLAB_R2018b.app/bin/matlab -desktop -r "matlab.engine.shareEngine('MATLAB_NFB_SPM_00001')" > /dev/null 2>&1 &
# uncomment for DCM feedback
/Applications/MATLAB_R2018b.app/bin/matlab -desktop -r "matlab.engine.shareEngine('MATLAB_NFB_MODEL_HELPER_00001')" > /dev/null 2>&1 &

```

Now I can move on to learning the software! 😄

**Comment by espdev** [comment]
Fixed in #90
